.ci/pipeline_integration_test

```diff
@@ -79,26 +86,87 @@ function setup_ginkgo() {
     echo "Successfully installed Ginkgo."
 }
 
-function fetch_control_kubeconfig() {
-    ${cli_path} config attribute --cfg-type kubernetes --cfg-name mcm-ci-aws-oot-control --key kubeconfig  > dev/control_kubeconfig.yaml
+# Fetch service account kubeconfig
+function fetch_service_account_kubeconfig() {
+    ${cli_path} config attribute --cfg-type kubernetes --cfg-name mcm-ci-garden-dev-virtual --key kubeconfig  > dev/sa_kubeconfig.yaml
+}
+
+function make_kubeconfig_request() {
+    kubectl --kubeconfig=dev/sa_kubeconfig.yaml create -f <(printf '{"spec":{"expirationSeconds":10800}}') --raw /apis/core.gardener.cloud/v1beta1/namespaces/"$1"/shoots/"$2"/adminkubeconfig | jq -r ".status.kubeconfig" | base64 -d > "$3"
```

Comment:
kindly make a variable for kubeconfig expiry time on top of file.. so that its easily configurable in future... you can even pass it as part of the arguments